docs/tutorial/security.md

```diff
@@ -279,19 +279,20 @@ security-conscious developers might want to assume the very opposite.
 
 ```js title='main.js (Main Process)'
 const { session } = require('electron')
+const URL = require('url').URL
 
 session
   .fromPartition('some-partition')
   .setPermissionRequestHandler((webContents, permission, callback) => {
-    const url = webContents.getURL()
+    const parsedUrl = new URL(webContents.getURL())
 
     if (permission === 'notifications') {
       // Approves the permissions request
       callback(true)
     }
 
     // Verify URL
-    if (!url.startsWith('https://example.com/')) {
+    if (!parsedUrl.startsWith('https://example.com')) {
```

Comment:
```suggestion
    if (parsedUrl.protocol !== 'https:' || parsedUrl.host !== 'example.com') {
```

Reply:
Thank  you for the heads up, I have applied your suggestion :^)